examples/awssns-quickstart/src/main/resources/META-INF/microprofile-config.properties

```diff
@@ -0,0 +1,13 @@
+# Configures AWS App public URL. This URL should be accessible by
+# AWS SNS Topic, it can be a public URL or a URL accessible by SNS
+# within the same VPC.
+sns-app-public-url=http://4befe2f5.ngrok.io
```

Comment:
we probably should let this entry `blank` and explain in the README what needs to be done.

Reply:
if the local SNS mock docker works we could replace it with localhost and explain it in README as you suggested.